ir_datasets/formats/argsme.py

```diff
@@ -0,0 +1,412 @@
+from datetime import datetime
+from enum import Enum
+from pathlib import Path
+from re import sub
+from typing import NamedTuple, List, Optional
+
+from ijson import items
+
+from ir_datasets.formats import BaseDocs
+from ir_datasets.indices import PickleLz4FullStore
+from ir_datasets.util import Cache, use_docstore
+
+
+class ArgsMeStance(Enum):
+    """
+    See the corresponding Java source files from the args.me project:
+    https://git.webis.de/code-research/arguana/args/args-framework/-/blob/master/src/main/java/me/args/argument/Stance.java
+    """
+    PRO = 1
```

Comment:
Maybe this should be an enum of `str`s instead? Same for the other enums below. The advantage would be that the user wouldn't need to import `ArgsMeStance` to do comparisons/filtering/etc -- they'd just need to compare `stance.value == "PRO"`.

I don't feel strongly about this, though. What do you think?

Reply:
I'd favor forcing users to do enum comparisons explicitly. But if string enums are more common in ir_datasets, that'd be fine too.